<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceDataset</class>
 <widget class="QMainWindow" name="FaceDataset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FaceDataset</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>721</width>
      <height>711</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Student Entry</string>
     </attribute>
     <widget class="QLineEdit" name="studentIDText">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>30</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="inputMask">
       <string>99999</string>
      </property>
     </widget>
     <widget class="QLabel" name="studentIDLabel">
      <property name="geometry">
       <rect>
        <x>53</x>
        <y>38</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Student ID:</string>
      </property>
     </widget>
     <widget class="QLabel" name="studentEmailLabel">
      <property name="geometry">
       <rect>
        <x>28</x>
        <y>127</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Student E-mail:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="studentNameText">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>75</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="studentEmailText">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>120</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="studentNameLabel">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>82</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Student Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="pictureLabel">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>0</y>
        <width>120</width>
        <height>120</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="addToDatasetButton">
      <property name="geometry">
       <rect>
        <x>558</x>
        <y>125</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add to dataset</string>
      </property>
     </widget>
     <widget class="QLabel" name="webCamLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>640</width>
        <height>480</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="addDatasetStatus">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>129</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>128</red>
            <green>128</green>
            <blue>128</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Course-Student Entry</string>
     </attribute>
     <widget class="QLabel" name="classLabel">
      <property name="geometry">
       <rect>
        <x>53</x>
        <y>14</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Course List</string>
      </property>
     </widget>
     <widget class="QLabel" name="studentLabel">
      <property name="geometry">
       <rect>
        <x>333</x>
        <y>14</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Student List</string>
      </property>
     </widget>
     <widget class="QListView" name="studentListView">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>341</width>
        <height>561</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="courseComboBox">
      <property name="geometry">
       <rect>
        <x>46</x>
        <y>37</y>
        <width>211</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="updateDatabaseButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>610</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Database</string>
      </property>
     </widget>
     <widget class="QLabel" name="updateDatabaseStatus">
      <property name="geometry">
       <rect>
        <x>358</x>
        <y>616</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>128</red>
            <green>128</green>
            <blue>128</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
